spec/lib/tasks/evm_settings_spec.rb

```diff
@@ -27,6 +27,7 @@
         expect(STDOUT).to receive(:puts).with("/authentication/oidc_enabled=false")
         expect(STDOUT).to receive(:puts).with("/authentication/provider_type=none")
         expect(STDOUT).to receive(:puts).with("/authentication/local_login_disabled=false")
+        expect(STDOUT).to receive(:puts).with("/prototype/messaging_type=")
```

Comment:
Can you add a value for this new type in the hash on lines 11-20 ☝️  so this test verifies this type just like the other ones?

Reply:
OK, WillCo 😁